### PR TITLE
Fix ie build

### PR DESCRIPTION
**Pull request by bluebill1049**
#2542

- [x] update the doc
- [x] update changelog
- [x] revert code

### CODE REVIEW

File: rollup/createRollupConfig.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ export function createRollupConfig(options) {
             drop_console: true,
           },
         }),
-      options.plugins,
+      ...(options.plugins || []),
```
Comment (kotarella1110) [inline rollup/createRollupConfig.js:67]: It doesn't seem to be minify.
Must be after rollup-plugin-typescript2 in the plugin list.

スクリーンショット 2020-08-11 12 04 11

Comment (bluebill1049) [reply rollup/createRollupConfig.js:67]: hmmm, so either way we need the user to install as dependency, I think it's fair for IE 11 support.

Comment (bluebill1049) [reply rollup/createRollupConfig.js:67]: doc update: https://github.com/react-hook-form/documentation/pull/405

Comment (bluebill1049) [reply rollup/createRollupConfig.js:67]: @kotarella1110 you happy to merge this PR?

Comment (kotarella1110) [reply rollup/createRollupConfig.js:67]: @bluebill1049 please wait. I will fix this.

Comment (kotarella1110) [reply rollup/createRollupConfig.js]: @bluebill1049 done. In this commit https://github.com/react-hook-form/react-hook-form/pull/2551/commits/9718436ee668da9a49522803d3d490164878dd7f, I fixed it to be minify when production build.

スクリーンショット 2020-08-11 23 09 36

Comment (bluebill1049) [reply rollup/createRollupConfig.js]: awesome! thanks @kotarella1110 merge it in.

**Pull request merged**